FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.JobManagement/AzureStorageClientFactory.cs

```diff
@@ -21,31 +21,34 @@ public class AzureStorageClientFactory : IAzureStorageClientFactory
         private readonly string _tableName;
         private readonly string _queueUrl;
         private readonly string _queueName;
+        private readonly string _internalConnectionString;
 
         public AzureStorageClientFactory(
-            IOptions<JobConfiguration> config,
+            IOptions<JobConfiguration> jobConfiguration,
```

Comment:
as `JobConfiguration` and `StorageConfiguration` are both for azure storage configuration, how about integrating them?

Reply:
Since 'StorageConfiguration' interface will not expose to customer, we'd better separate them.